### ISSUE TITLE
v2 - Issues with PSUScript cmdlets. 

### ISSUE THREAD

**Issue by adamdriscoll** [ironmansoftware/powershell-universal#193]
https://forums.ironmansoftware.com/t/testing-psu-v2-0-0-nightly-issues/4994

**Comment by rjvrijn** [comment]
Looks like this issue still exists in version 2.4.1 and 2.5.0 nightly

**Comment by adamdriscoll** [comment]
I just tried these on the latest 2.5 build. 

This is expected when a script doesn't exist. It no longer returns a 500 but rather a 404 not found. 

```
Get-PSUScript -name Test123.ps1

Nov 7, 2021 8:51 AM [error] Call failed with status code 404 (Not Found): GET http://localhost:5000/api/v1/Script/Test123.ps1 
```

As for New-PSUScript, I can reproduce this and has been fixed in 2.5 as of today.
